### ISSUE TITLE
Cannot build Mockito on Windows 64b with Java 8 

### ISSUE THREAD

**Issue by scais** [mockito/mockito#765]
Hi guys,

 I am trying to build Mockito on my Windows 64 machine with jdk1.8.0_66 and the build is failing. I tried the `master `branch and also the `release/2.x` branch, both are having the same problem. 

When I run `gradlew clean build` I am getting:

```
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: unclosed character literal
        mock.oneArg(Character.valueOf('Ôé¼'));
                                      ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: illegal character: '\u201a'
        mock.oneArg(Character.valueOf('Ôé¼'));
                                        ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: illegal character: '\u00ac'
        mock.oneArg(Character.valueOf('Ôé¼'));
                                         ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: unclosed character literal
        mock.oneArg(Character.valueOf('Ôé¼'));
                                          ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: ';' expected
        mock.oneArg(Character.valueOf('Ôé¼'));
                                            ^
5 errors
:compileTestJava FAILED
:removeTestResources

FAILURE: Build failed with an exception.
```

After removing the line containing `        mock.oneArg(Character.valueOf('€'));` inside **MatchersTest.java** on line 599, the build works OK.

Anyone else facing similar trouble?

**Comment by TimvdLippe** [comment]
On Linux this is working fine. I thought that @bric3 (who made the test in https://github.com/mockito/mockito/commit/c9e8562c994d23286d326a8ee19e2598d36ebfda) is on a Mac. I suppose we can alter the value just fine for Windows. We do not build our tests on Windows ourselves, so thanks for trying it out!

**Comment by bric3** [comment]
This looks like a character encoding issue.
The last time I checked on Windows there was nothing wrong. I'll recheck when I have access to a windows box.

**Comment by scais** [comment]
Thanks guys! FYI: Just checked on different Windows 64 and build fails there as well.

**Comment by raphw** [comment]
Just checked on a Windows VM and the problem does not happen here. Strange.

**Comment by scais** [comment]
I managed to build it on my Windows 7 64B machine by using `gradlew build -Dfile.encoding=UTF-8`

How I found out:

According to http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html the Java compiler javac uses the default system encoding if the `-encoding` is not specified. My system default encoding in the machine is `Cp1252`, retrieved as a result of gradle task:

```
task printSystemEncoding << {
    println System.getProperty("file.encoding")
}
```

Another solution what worked for me was to configure directly the `javac` task to use the `encoding` flag with utf8:

```
tasks.withType(JavaCompile) {
    options.encoding = 'UTF-8'
}
```

The problem is probably that the file `MatchersTest.java` is saved in encoding which is not compatible with the default Windows `Cp1252`. I would like to **suggest** adjustment of the test / providing of default encoding set to utf-8 inside the build.
